src/js/pages/Escrow/components/CardEscrowSeller.jsx

```diff
@@ -167,43 +167,44 @@ class CardEscrowSeller extends Component {
       showFundButton = false;
     }
 
-    if(arbitrationDetails && (arbitrationDetails.open || arbitrationDetails.result.toString() !== "0")){
-      return (
-        <Card>
-          <CardBody className="text-center p-5">
-            <Dispute />
-          </CardBody>
-        </Card>
-      );
-    }
-
-
-    if(showFundButton) step = 2;
-    if(fundStatus === States.pending || (trade.mining && trade.status === escrow.helpers.tradeStates.waiting)) step = 3;
-    if(fundStatus === States.success) step = 4;
-    if(releaseStatus === States.pending || (trade.mining && (trade.status === escrow.helpers.tradeStates.funded || trade.status === escrow.helpers.tradeStates.paid))) step = 5;
-    if(releaseStatus === States.success || trade.status === escrow.helpers.tradeStates.released) step = 6;
+    if (showFundButton) step = 2;
+    if (fundStatus === States.pending || (trade.mining && trade.status === escrow.helpers.tradeStates.waiting)) step = 3;
+    if (fundStatus === States.success) step = 4;
+    if (releaseStatus === States.pending || (trade.mining && (trade.status === escrow.helpers.tradeStates.funded || trade.status === escrow.helpers.tradeStates.paid))) step = 5;
+    if (releaseStatus === States.success || trade.status === escrow.helpers.tradeStates.released) step = 6;
+    if (arbitrationDetails && arbitrationDetails.open && arbitrationDetails.result.toString() === "0") step = 10;
+    if (arbitrationDetails && arbitrationDetails.result.toString() !== "0") step = 11;
 
     let component;
-    switch(step){
+    switch (step) {
+      case 10:
```

Comment:
Not for this PR, but I'm thinking that we need to move away from this switch to something more robust

Reply:
Agreed.
I tried changing it, but it's a lot more massive than I anticipated, so I reverted. 
We need a PR just for that, because there's a lot of possibility of breaking it